.github/workflows/firebase-deploy.yml

```
@@ -25,10 +25,13 @@ jobs:
         run: npm run build
 
       - name: Deploy to Firebase
```

actionlint (1.7.4)

step must run script with "run" section or run action with "uses" section

(syntax-check)